web/control_flow/using_conditional_templates/my_element.html

```diff
@@ -1,15 +1,15 @@
-<polymer-element name="my-element">
+<dom-module id="my-element">
   <template>
-    <template if="{{showAnswer}}">
+    <template is="dom-if" if="{{showAnswer}}">
       <div>42</div>
-      <button on-tap="{{toggleView}}">Show question</button>
+      <button on-tap="toggleView">Show question</button>
```

Comment:
lets move this button below the `dom-if` templates, and then bind its text to a computed binding. 

```html
<dom-module id="my-element">
  <template>
    <template is="dom-if" if="{{showAnswer}}">
      <div>42</div>
    </template>
    <template is="dom-if" if="{{!showAnswer}}">
      <div>
        What is the answer to "The Ultimate Question of Life, the
        Universe, and Everything"?
      </div>
    </template>
    <button on-tap="toggleView">{{buttonText(showAnswer)}}</button>
  </template>
</dom-module>
```

```dart
@HtmlImport('my_element.html')
library my_element;

import 'package:web_components/web_components.dart' show HtmlImport;
import 'package:polymer/polymer.dart';

@PolymerRegister('my-element')
class MyElement extends PolymerElement {
  @property bool showAnswer = false;

  MyElement.created() : super.created();

  @reflectable
  void toggleView([_, __]) {
    set('showAnswer', !showAnswer);
  }

  @reflectable
  String buttonText(_) => 'Show ${showAnswer ? 'question' : 'answer'}';
}
```